src/EditorFeatures/Core/GoToDefinition/AbstractGoToDefinitionService.cs

```diff
@@ -41,12 +41,30 @@ internal abstract class AbstractGoToDefinitionService : IGoToDefinitionService
 
         public bool TryGoToDefinition(Document document, int position, CancellationToken cancellationToken)
         {
-            // First try to compute the referenced symbol and attempt to go to definition for the symbol.
+            ISymbol symbol = null;
             var symbolService = document.GetLanguageService<IGoToDefinitionSymbolService>();
-            var (symbol, span) = symbolService.GetSymbolAndBoundSpanAsync(document, position, cancellationToken).WaitAndGetResult(cancellationToken);
-            if (symbol == null)
+
+            // If the cursor is on the keyword for overriding, we'll find the overridden symbol
+            var syntaxTree = document.GetSyntaxTreeAsync(cancellationToken).WaitAndGetResult(cancellationToken);
+            var syntaxService = document.GetLanguageService<ISyntaxFactsService>();
```

Comment:
`IGoToDefinitionSymbolService` is already language-specific -- why not put the language specific stuff in those?

Reply:
Also: it may be possible to push this farther down into the system so then clicking on "overrides" also does highlight references, and highlights the overrides as a reference.

Reply:
The reason I put it in `ISyntaxFactsService` is because the logic is purely syntactic (no symbols involved). 
I'm happy to move to `IGoToDefinitionSymbolService` (let me know).

Reply:
Well, right now AbstractToGoDefinitionService is more or less opaque with the concept of where the symbol comes from: it's job is to get the symbol and pass it around. "What symbol?" is a question for that other layer.